README.md

```diff
@@ -141,6 +142,21 @@ const client = new Version2Client({
 });
 ```
 
+##### [PAT](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html)
+
+```typescript
+import { Version2Client } from 'jira.js';
+
+const client = new Version2Client({
+  host: 'https://your-domain.atlassian.net',
+  authentication: {
+    pat: {
```

Comment:
I think better to do `pat` -> `personalAccessToken` and change type from object to string